src/puppetlabs/jdbc_util/core.clj

```diff
@@ -139,7 +140,9 @@
       (let [tail (.substring s (.length ^String match) (.length s))
             replaced (str/replace match #"(.*)\?$" (str "$1" replacement))]
         (str replaced tail))
-      (throw (IllegalArgumentException. (str "there are not " n " '?'s in the given string"))))))
+      (throw (IllegalArgumentException. (trun "There is not one '?' in the given string"
```

Comment:
I think the intent here would be a little clearer if the singular version were "There are no '?'s in the given string"